### DISCUSSION TITLE
how to get client login mode ?

### DISCUSSION THREAD

**Discussion by q16695** [games647/FastLogin#835] [General]
how to get client login mode ?

**Comment by TuxCoding** [comment]
What do you mean with client login mode?

**Reply by q16695** [reply]
if a player use the cracked,server send a message and the player return the login mode (cracked)

**Reply by TuxCoding** [reply]
See the FAQ, that's not possible. The disconnect message is not server side, it's from the client.

**Reply by q16695** [reply]
Although you doesn't solve my problem, I still want to thank you for answering me
